### DISCUSSION TITLE
webui.call triggers a browser error saying webui undefined,

### DISCUSSION THREAD

**Discussion by Kremez** [webui-dev/webui#202] [Q&A]
If I want to call a C function from Javascript I have been trying to use webui.call as described in the documentation.  However, I get an error saying webui is undefined.  How do I properly define webui so I can use it in javascript?

Thanks,
Simon

**Comment by hassandraga** [comment]
You are probably using the old version `2.3.0` (_Which has the old webui_fn()_). Or you are using an old beta `2.4.0` (_Which webui js class instance was not global yet_).

I suggest downloading the latest 2.4.0 auto build:
https://github.com/webui-dev/webui/actions

**Comment by Kremez** [comment]
I will wait until 2.4.0 is the stable release.

**Reply by hassandraga** [reply]
We are finishing the auto-build scripts, makefiles and other staff to be ready for the release. The core code is finished. However, you can wait, as someone may report an issue or request a feature.

**Reply by Kremez** [reply]
So, when will this latest version 2.4.0 be available for download?

**Reply by hassandraga** [reply]
Since there is no `bug` report in the opened issues, I guess we can release it this weekend.

**Reply by Kremez** [reply]
Hi there! Hope all is well with webui 2.4.0. Please tell me when I can expect this version to be released?

**Reply by hassandraga** [reply]
Yesterday we fixed the last known issue. We should now test the library a few more times to make sure.

**Reply by ttytm** [reply]
👍 I think so to that the release can be done very soon. As it's to expect that the library won't change much anymore util the release, you can already start using the nightly releases. They include the main changes of the upcoming 2.4.0 and are already more tested and contain bug fixes that 2.3.0 doesn't have. Then, when 2.4.0 is out it should be fairly seamless switching to the stable version.

**Reply by hassandraga** [reply]
I hope we can release it this weekend. I want to start a new AI-GUI project using WebUI `v2.4.0`.

**Reply by Kremez** [reply]
Thanks guys, you are doing a great job, it helps a lot of developers.
